pkgs/applications/editors/vscode/default.nix

```diff
@@ -87,6 +87,11 @@ in
     dontBuild = true;
     dontConfigure = true;
 
+    patchPhase = ''
```

Comment:
You should use `postPatch` or else things in `patches` won't be applied, if the need ever comes.